zadania/Program.cs

```diff
@@ -3,25 +3,15 @@
 int width = 20;
 int height = 20;
 int n = 15000;
-double[,] generateTableDensity(int width, int height)
+double[,] generateTable(int x, int y, Func<int, int, double> generator)
 {
-    double[,] densityTable = new double[width, height];
-    for (int x = 0; x < width; x++)
+    double[,] Table = new double[x,y];
+    for (int i = 0; i < width; i++)
```

Comment:
Jeśli podajesz w parametrach funkcji wymiary tabel to korzystaj z nich, a nie z global scope width i height. Unikniesz tym samym błędów związanych z wielokrotnym wykorzystaniem tej samej funkcji w różnych miejscach